src/rpc/net.cpp

```diff
@@ -224,14 +224,14 @@ static RPCHelpMan getpeerinfo()
             // addnode is deprecated in v0.21 for removal in v0.22
             obj.pushKV("addnode", stats.m_manual_connection);
         }
-        obj.pushKV("startingheight", stats.nStartingHeight);
         if (fStateStats) {
             if (IsDeprecatedRPCEnabled("banscore")) {
                 // banscore is deprecated in v0.21 for removal in v0.22
                 obj.pushKV("banscore", statestats.m_misbehavior_score);
             }
             obj.pushKV("synced_headers", statestats.nSyncHeight);
             obj.pushKV("synced_blocks", statestats.nCommonHeight);
+            obj.pushKV("startingheight", statestats.m_starting_height);
```

Comment:
In 2f023a8c0e445985a20, `startingheight` should remain before `synced_headers` and `synced_blocks` (here and in the help) for consistency with the GUI peers info (or maybe less preferably, the GUI should be updated to follow this change)

Reply:
done